packages/prettier-plugin/test/comments.test.ts

```diff
@@ -0,0 +1,103 @@
+import { format } from "./helpers.js";
+
+describe("comments", () => {
+  test("line comment", async () => {
+    expect(
+      await format(`// line comment
+123`)
+    ).toBe(`// line comment
+123`);
+  }, 20000); // first call to prettier is slow and occasionally causes timeouts in Github Actions
+
+  test("comment indentation is ignored", async () => {
+    expect(
+      await format(`   // line comment
```

Comment:
I wonder if there's a better way of organizing these variables - the indentation here makes them harder to read